### ISSUE TITLE
Mention the usage of Polly in install instructions

### ISSUE THREAD

**Issue by ScholliYT** [andrewlock/PwnedPasswords#23]
I didn't knew polly was needed for this: https://github.com/andrewlock/PwnedPasswords/blob/master/samples/PwnedPasswords.Sample.NetCore3/Startup.cs#L15

**Comment by SeanFarrow** [comment]
Does it not get restored from NuGet?
I’ll take a look at the master branch tomorrow to see what is going on.

**Comment by ScholliYT** [comment]
At least for the `PwnedPasswords.Client` I had trouble getting it running. Things like `AddTransientHttpErrorPolicy` are polly specific. 
Now I just run without Polly required as NuGet. But I also only got in my ConfigureServices
```csharp
identityBuilder.AddPwnedPasswordValidator<ApplicationUser>();
```
and the two package references in .csproj file. 

Is the behaviour of retry and timeout implementet by the `PwnedPasswords.Client` by default? Without polly? Or would I need to set things up as explained in the Readme if I wanted that?

Just to clarify: I'm not trying to run the example mentioned with the link above. I was trying to implement this in my application.

**Comment by SeanFarrow** [comment]
The PwnedPasswordsClient constructor takes in an HttpClient and calls it’s GetAsync method to determine whether the passed in password has been pwned.
It does not by itself handle retries, so you would need to use the Polly integration with HttpClientFactory for that as the sample does.

So, to use the PwnedPasswordsClient, you don’t strictly need Polly, but I always tent to use it as a matter of course when doing any HTTP work, particularly if I’m going to want to do things like retry, cache returned values and/or provide a fallback mechanism.
The only exception to this is if an SDK I’m using wraps the HTTP calls, good examples of this are the ones available for AWS and Azure.

**Comment by ScholliYT** [comment]
Thanks for the explanation. I would appreciate to see this mentioned in the Readme. As you said the retry mechanism is really handy 😄

**Comment by SeanFarrow** [comment]
Yep, that’s a good shout, I’ll do this when I get a spare half hour, probably in the next few weeks.
